README.md

```diff
@@ -186,6 +186,12 @@ include ':app'
 
 </details>
 
+<details>
```

Comment:
I don't think this is needed in `Linking`, as I already added a section at the bottom to the README mentioning this.

Could you possibly take a look at https://github.com/rebeccahughes/react-native-device-info#react-native-web and mention "react-native-dom" in it? (feel free to rename the section title)

Reply:
That section must've gotten pulled in from the merge.  I'll remove it and add rn-dom to the bottom.